tests/phpunit/includes/utils.php

```diff
@@ -645,3 +645,45 @@ function test_rest_expand_compact_links( $links ) {
 	}
 	return $links;
 }
+
+/**
+ * Removes select handles from $wp_scripts.
+ */
+function unregister_all_script_handles() {
+	global $wp_scripts;
+
+	// Do not deregister following libraries through this function.
+	$libraries = array(
+		'jquery',
+		'jquery-core',
+		'jquery-migrate',
+		'jquery-ui-core',
+		'jquery-ui-accordion',
+		'jquery-ui-autocomplete',
+		'jquery-ui-button',
+		'jquery-ui-datepicker',
+		'jquery-ui-dialog',
+		'jquery-ui-draggable',
+		'jquery-ui-droppable',
+		'jquery-ui-menu',
+		'jquery-ui-mouse',
+		'jquery-ui-position',
+		'jquery-ui-progressbar',
+		'jquery-ui-resizable',
+		'jquery-ui-selectable',
+		'jquery-ui-slider',
+		'jquery-ui-sortable',
+		'jquery-ui-spinner',
+		'jquery-ui-tabs',
+		'jquery-ui-tooltip',
+		'jquery-ui-widget',
+		'backbone',
+		'underscore',
+	);
+
+	foreach ( $wp_scripts->registered as $handle_name => $handle ) {
+		if ( ! in_array( $handle_name, $libraries, true ) ) {
+			wp_deregister_script( $handle_name );
+		}
+	}
+}
```

Comment:
I don't think we need this function. I tried disabling it and all tests still passed.

```suggestion
```

Reply:
@westonruter yes we can remove it. It was introduced to clear the enqueues from previous test cases. 
Most of the script handles from new testcases were later renamed to have a namespace

However, we need to double check for test group `media` after the removal.


**Adding a note here as why this was introduced in the first place:**
`get_echo( 'wp_print_scripts' )` just clears the display buffer but doesn't `wp_deregister_script` any script. 
Scripts from previous testcases still remains enqueued as we use common global. 
This can cause an issue when you run a testcase and use pre-existing handle name (existing from previous tests), and the order in which script is loaded matters.